specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json

```diff
@@ -15,6 +15,527 @@
+            "$ref": "#/parameters/ArchiveNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly."
+          },
+          "202": {
+            "description": "The request was successful; the operation will complete asynchronously.",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "204": {
+            "description": "The archive does not exist in the subscription."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ArchiveDelete": {
+            "$ref": "./examples/ArchiveDelete.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      },
+      "patch": {
+        "tags": [
+          "Archives"
+        ],
+        "description": "Updates a archive for a container registry with the specified parameters.",
+        "operationId": "Archives_Update",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v4/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v4/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v4/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/RegistryNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PackageTypeParameter"
+          },
+          {
+            "$ref": "#/parameters/ArchiveNameParameter"
+          },
+          {
+            "name": "archiveUpdateParameters",
+            "in": "body",
+            "description": "The parameters for updating a archive.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ArchiveUpdateParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the operation has completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/Archive"
+            }
+          },
+          "201": {
+            "description": "The request was successful; the operation will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/Archive"
+            },
+            "headers": {
+              "Azure-AsyncOperation": {
+                "type": "string"
+              }
+            }
+          },
```

Comment:
LRO patch should use 200 and 202:
[https://github.com/Azure/azure-openapi-validator/blob/main/docs/patch-response-code.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/patch-response-code.md)